src/ui/jexlParser.js

```diff
@@ -0,0 +1,176 @@
+import Lexer from "mozjexl/lib/Lexer";
```

Comment:
This should be a typescript file.

I'd like to see types for the various AST shapes so we can be type safe here.